phoenix-pherf/src/test/resources/scenario/test_scenario_with_load_profile.xml

```diff
@@ -0,0 +1,362 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I prefer to put this scenario file under /phoenix-pherf/src/main/resources/scenario/ dir since this is an example of how to run instead of a test case.

Reply:
@yanxinyi This file is used in testWorkloadWithLoadProfile in ConfigurationParserTest. So I think it makes sense to keep it in the test/resources folder.